src/providers/ConnectorState/LogLoop.ts

```diff
@@ -1,12 +1,16 @@
 import Minilog from '@cozy/minilog'
-import CozyClient from 'cozy-client'
 import uniqueId from 'lodash/uniqueId'
 
-import {
-  clearLogs,
-  spliceLogs
-} from '/providers/ConnectorState/ConnectorLogsSlice'
+import { removeLogs } from '/providers/ConnectorState/ConnectorLogsSlice'
 import { store } from '/store'
+import { default as CozyClientType } from 'cozy-client'
+
+interface CozyClient extends CozyClientType {
+  getStackClient: () => {
+    uri: string
+    fetchJSON: <T>(method: string, path: string, body: unknown) => Promise<T>
+  }
+}
```

Comment:
Should we create this interface elsewhere in the project in order to be able to reuse it easily? Don't we already have a place to store these interfaces? @acezard @Ldoppea